### PR TITLE
add dark theme and some bug fixes

### PR DESCRIPTION
**Pull request by Nexite**

### CODE REVIEW

File: server_release_template/web_gui/js/app/settings.js [hunk 1]
```diff
@@ -388,6 +388,9 @@ define([
                     case "controllerMode":
                         addDropdown(element, path, name, advanced)
                         break;
+                    // case "themeDropdown":
+                    //     addDropdown(element, path, name, advanced)
+                    //     break;
```
Comment (zarik5) [inline server_release_template/web_gui/js/app/settings.js:393]: Delete commented code

File: alvr/server_driver/cpp/alvr_server/Settings.h [hunk 2]
```diff
@@ -128,5 +128,7 @@ class Settings
 	bool m_captureComposedDDSTrigger = false;
 	
 	int m_controllerMode = 0;
+
+	std::string m_appearance;
```
Comment (zarik5) [inline alvr/server_driver/cpp/alvr_server/Settings.h:132]: We don't need this here

File: server_release_template/web_gui/js/app/customSettings.js [hunk 3]
```diff
@@ -560,6 +560,69 @@ define([
...
+            }
+            var bootstrap = $("#bootstrap");
+            var selectal = $("#selectal");
+            var style = $("#style");
+            
+            var themeSelector = $("form#_root_extra_theme-choice-").first();
+            var themeColor = $("input[name='theme']:checked").val();
+            // $("label[for='_root_extra_theme_systemDefault-choice-']").text("System");
+            // $("label[for='_root_extra_theme_classic-choice-']").text("Classic");
+            // $("label[for='_root_extra_theme_darkly-choice-']").text("Darkly");
```
Comment (zarik5) [inline server_release_template/web_gui/js/app/customSettings.js:578]: Delete commented code

**Review by zarik5 — CHANGES_REQUESTED** [review]
Some small nits, otherwise looks good.

**Comment by Nexite** [comment]
@zarik5 fixed

**Pull request merged**